src/django/api/admin.py

```
@@ -435,6 +440,76 @@ class USCountyTigerlineAdmin(admin.ModelAdmin):
+            obj.status = PartnerDataFileUpload.Status.PROCESSING
+            obj.save(
+                update_fields=["batch_job_id", "status", "updated_at"]
+            )
+            messages.success(
+                request,
+                (
+                    "Partner data file was queued for moderation ingestion. "
+                    f"Batch job ID: {job_id}"
+                ),
+            )
+        except Exception as error:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)